.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,21 +1,58 @@
+fix: Fix build, language, links, or metadata.
+-->
 
+<!-- Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR. -->
 
-### What issues does this PR fix or reference?
+## What does this PR do?
 
+## What issues does this PR fix or reference?
 
-### Specify the version of the product this PR applies to.
+## Specify the version of the product this PR applies to
 
+## Pull Request Checklist for committers and reviewers
 
-### PR Checklist
+The content of this Pull Request is compliant with this checklist:
 
-As the author of this Pull Request I made sure that:
+### Level 1 (automated tests blocking PR merge)
 
-- [ ] `vale` has been run successfully against the PR branch
-- [ ] Link checker has been run successfully against the PR branch
-- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
-- [ ] Changed article references are updated where they are used (or a redirect has been configured on the docs side):
+- [ ] *`Build and validate PR / link checker (pull_request)`* is successful
+  - [ ] *`Build using Antora`* step is successful: documentation is building without warnings, and all AsciiDoc attributes have a definition.
+  - [ ] *`Validate links using htmltest`* step is successful: internal and external links are valid.
+  - [ ] *`Validate language on files added or modified`* step is successful and reports no vale errors: basic language validation, no major deviations from [IBM Style Guide](https://www.oreilly.com/library/view/the-ibm-style/9780132118989/), [Red Hat Supplementary Style Guide for product documentation](https://redhat-documentation.github.io/supplementary-style-guide/), and project specific language rules.
+- [ ] *`eclipsefdn/eca`*  is successful: all contributors have signed the Eclipse Contributor Agreement  (ECA).
+
+### Level 2
+
+- [ ] Any procedure:
+  - [ ] Successfully tested.
+  - [ ] Describes a scenario already covered by QE tests, otherwise Che QE team has acknowledged an issue.
+- [ ] Any page or link rename:
+  - [ ] The page contains a redirection for the previous URL.
+  - [ ] Propagate the URL change in:
     - [ ] Dashboard [default branding data](https://github.com/eclipse-che/che-dashboard/blob/main/src/services/bootstrap/branding.constant.ts)
     - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)
+- [ ] *`Build and validate PR / link checker (pull_request)`* reports no warnings.
+  - [ ] *`Validate language on files added or modified`* step reports no vale warnings.
+- [ ] Content is appropriate for the intended audience.
+- [ ] Effective titles.
+- [ ] Information is clear and concise, no ambiguity.
+- [ ] No sentences with low information value.
+- [ ] Sequence of information is logical.
+
+### Level 3
+
+- [ ] Builds locally, and the output looks fine.
+- [ ] Consistent with the rest of the docs.
```

Comment:
This part is not clear for me

Reply:
Language is consistent with the rest of the docs